src/renderer/components/archive/archive-search.js

```diff
@@ -184,6 +184,10 @@ class ArchiveSearch extends PureComponent {
         this.openEntry(result.sourceID, result.entry);
       }
     }
+
+    if (e.keyCode === 27) {
+      this.closeSearch();
```

Comment:
@sallar does it also close on unfocus? Would be good if it was also possible via mouse.

Reply:
Yeah if you click out or press esc it closes the box

Reply:
👍